app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -31,7 +31,7 @@ def selected_location_hours(prefix)
       end
 
       def needs_proof_of_address?
-        !(enrollment.current_address_matches_id || enrollment.capture_secondary_id_enabled)
+        !enrollment.current_address_matches_id
```

Comment:
This is incorrect based on the original logic. We're setting `enrollment.capture_secondary_id_enabled` to always be `true` in this situation, so here's how the logic simplifies.

Original:
```ruby
!(enrollment.current_address_matches_id || enrollment.capture_secondary_id_enabled)
```

Updated with `true`:
```ruby
!(enrollment.current_address_matches_id || true)
```

The OR (`||`) part will always be true since the right is true, so this is equivalent:
```ruby
!(true)
```

This means the correct refactor would always return `false` for this value. We may also be able to remove the conditional from the views and remove this method altogether.
```suggestion
        false
```

Reply:
ahh thank you for that explanation! i agree if it's always false we likely no longer need it.  Want to double check the impact this will have to the emails though - we do check this value in the ready to verify email and if it's true we have bullet point #4 which lists acceptable supporting docs to bring re: address. But it seems that we do allow someone to proof with a secondary id for a single reason -> Here we have a secondary id that should allow someone to proof successfully (though it's unclear to me what ID a postal worker would see in this case). https://github.com/18F/identity-idp/blob/460ab1428b8dd7555db9653c95194fc66404fa96/app/jobs/get_usps_proofing_results_job.rb#L13

Reply:
This feature is meant to replace the secondary ID check, so it makes sense to me that the bullet point may never appear when this feature is enabled.

Reply:
ok i'll go ahead and update it